.gitmodules

```diff
@@ -4,4 +4,4 @@
 	branch = develop
 [submodule "test/tests"]
 	path = test/jsontests
-	url = https://github.com/ethereum/tests.git
+	url = https://github.com/jwasinger/tests.git
```

Comment:
no thats not the way. 
just do
```
cd cpp-ethereum/test/jsontests
git checkout   branchintestrepo
```

do you have rights to open new branches at cpp/tests repository ?